pyrobosim/examples/demo_pddl.py

```diff
@@ -86,7 +86,7 @@ def start_planner(world, args):
     plan = planner.plan(
         robot,
         goal_literals,
-        focused=True,
+        apative=True,
```

Comment:
This should be `adaptive` and I fixed it in #60, which we can directly merge into here first.